---
layout: default
title: Research
author: Sohrab Shah
---

# Research

## Overview

Why do some cancer patients respond to treatment, while others succumb to their disease?  Why are some treatments effective initially, but fail over time?  How do cancer cells acquire the ability to spread from one part of the body to another?  These are the fundamental and unresolved questions which motivate cancer research worldwide.  Viewing cancer progression through the lens of evolutionary theory, our approach to addressing these problems centers on studying the genomes of cancer cells as fundamental units of information encoding biological properties.  We have an active research program across several interrelated major topics including: Cancer evolution, single cell genomics, mutational signatures and prediction of drug response. The primary activities in the lab consist of experimental design, large-scale cancer (epi)genomics/transcriptome data analytics, development of machine learning and Bayesian statistics methods and biological study of ovarian, breast and lymphoid cancers.   

## Cancer evolution

Our lab is motivated by studying cancer through the lens of evolution.  We are engaged in several studies that span both temporal and spatial multi-sample studies of our cancers of interest.  Observing the dynamics of genomically-defined clones reflected in timeseries biopsies of patient tumours, patient-derived xenografts, or through spreading of clones across anatomical sites is a key area of interest for our lab.  For example in recent work, we have identified clonal expansion patterns underpinning histological transformation in follicular lymphoma, mapped the spread of clones within the peritoneal cavity of ovarian cancers and identified reproducible clonal dynamics in patient derived breast cancer xenografts.   Our current questions relate to drug selection and the interplay between malignant cells and the tumour microenvironment.  

Relevant recent work: [link] [link] [link] [link]

## Single cell genomics of cancer

The unit of evolutionary selection in cancer is the cell.  Extraordinary progress in measurement technologies has made it possible to reliably and accurately sequence the genomes of individual cancer cells at scale.  We have recently optimized biophysical techniques and hidden Markov model approaches to ascertain highly accurate copy number profiles of thousands of cancer cells.  As such, studying the ‘population genetics’ of cancer cells is a tractable goal.  We are developing phylogenetics and fitness computational models through measuring the population dynamics of thousands of individual cancer cells across timeseries, spatial samples and in the presence of genetic and pharmacologic intervention.  These experiments and computational methods are improving our knowledge of background mutation rates, properties of positive and negative selection (not observable in bulk samples) and how phylogenetic topologies reflect the relative fitness of clones. Furthermore, as we integrate multi-modal measurements, profiling the evolving malignant population in the context of its tumour microenvironment will be a strong interest of the lab.

Relevant recent work: [link] [link] [link] [link]

## Mutational signatures in DNA repair deficient cancers

We recently published a landmark study showing how the genomes of ovarian cancer histotypes reflect the DNA repair abnormalities they harbour.  We are interested in how to optimize the computational discovery of genome-wide structural and point mutational signatures and how signatures can identify treatment opportunities for ovarian and breast cancers.  This work is being carried out at bulk and single cell resolution.  In addition, we are working in translation capacity to develop a robust genome-wide test to stratify ovarian cancers in the clinic.

Relevant recent work: [link] [link] [link] [link]

## Selection and drug response

"Making predictions is hard, especially about the future" - Nils Bohr (and many other variants attributed to other famous people: Mark Twain, Nostradamus, Yogi Berra, etc…) .  We have a keen interest in learning fitness trajectories from timeseries study of cancer populations within controlled interventions such as CRISPR or pharmacologic methods as a means to predict response to drugs.  Using extensions of population genetics theory, we are interested in predicting how cell populations will respond in the presence of a perturbation.  This is indeed ‘hard’ and entails the need to decipher stochastic drift, clonal interaction and positive selection.  Furthermore, drug response may not be encoded in the genome, requiring dynamic state switching through the epigenome and reflected in the transcriptome.  What proportion of drug response can be explained through encoded mutations in the genome?  This remains unknown. We are pursuing integrative, multimodal molecular views over time in bulk tissues and single cells as  substrate to address this question.  

## Bioinformatics and software development 

We invest heavily in developing robust analytical software for both internal and external use with broad distribution through open source repositories.  

# Key Collaborators:

Vancouver: Sam Aparicio, David Huntsman, Jessica McAlpine, Carl Hansen, Alexandre Cote-Bouchard, Marco Marra
Canada: Lincoln Stein, Paul Boutros, Anne-Marie Mes-Masson, Morag Park, Amit Oza
International: Greg Hannon, Carlos Caldas, Ravi Pandya

# Funding sources:

BC Cancer Foundation, CIHR, CCSRI, TFRI, CRUK, SU2C, CFI, Genome Canada, Genome BC, CANARIE, Microsoft, Canada Research Chairs, Michael Smith Foundation for Health Research


