---
layout: paper
title: "ddClone: joint statistical inference of clonal populations from single cell and bulk tumour sequencing data."
image: /assets/images/papers/28249593.png
authors: Sohrab Salehi, Adi Steif, Andrew Roth, Samuel Aparicio, Alexandre Bouchard-Côté, Sohrab P Shah
ref: Salehi et al. 2017. Genome Biol..
journal: "Genome biology <b>18</b>, 44 (2017)"
pdf: /assets/pdfs/papers/28249593.pdf
doi: 10.1186/s13059-017-1169-3
abbrev: "Genome Biol. <b>18</b>, 44 (2017)"
year: 2017
---

<br />
<div data-badge-popover="right" data-badge-type="donut" data-pmid="28249593" data-hide-no-mentions="true" class="altmetric-embed"></div>

# Abstract

Next-generation sequencing (NGS) of bulk tumour tissue can identify constituent cell populations in cancers and measure their abundance. This requires computational deconvolution of allelic counts from somatic mutations, which may be incapable of fully resolving the underlying population structure. Single cell sequencing (SCS) is a more direct method, although its replacement of NGS is impeded by technical noise and sampling limitations. We propose ddClone, which analytically integrates NGS and SCS data, leveraging their complementary attributes through joint statistical inference. We show on real and simulated datasets that ddClone produces more accurate results than can be achieved by either method alone.

